apps/vr-tests/src/stories/Sticky.Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,245 @@
+                constrainMode={ConstrainMode.unconstrained}
+                onRenderDetailsHeader={onRenderDetailsHeader}
+                onRenderDetailsFooter={onRenderDetailsFooter}
+                selection={this._selection}
+                selectionPreservedOnEmptyClick={true}
+                ariaLabelForSelectionColumn="Toggle selection"
+                ariaLabelForSelectAllCheckbox="Toggle selection for all items"
+                // tslint:disable-next-line:jsx-no-lambda
+                onItemInvoked={item => alert(`Item invoked: ${item.name}`)}
+              />
+            </MarqueeSelection>
+          </ScrollablePane>
+        </Fabric>
+      </div>
+    );
+  }
+}
+
+function onRenderDetailsHeader(props: IDetailsHeaderProps, defaultRender?: IRenderFunction<IDetailsHeaderProps>): JSX.Element {
+  return (
+    <Sticky stickyPosition={StickyPositionType.Header} isScrollSynced={true}>
+      {defaultRender!({
+        ...props,
+        onRenderColumnHeaderTooltip: (tooltipHostProps: ITooltipHostProps) => <TooltipHost {...tooltipHostProps} />
+      })}
+    </Sticky>
+  );
+}
+
+function onRenderDetailsFooter(props: IDetailsFooterProps, defaultRender?: IRenderFunction<IDetailsFooterProps>): JSX.Element {
+  return (
+    <Sticky stickyPosition={StickyPositionType.Footer} isScrollSynced={true} >
+      <div style={{ display: 'inline-block' }}>
+        <DetailsRow
+          columns={props.columns}
+          item={{
+            key: 'footer',
+            test1: 'Total 1',
+            test2: 'Total 2',
+            test3: 'Total 3',
+            test4: 'Total 4',
+            test5: 'Total 5',
+            test6: 'Total 6'
+          }}
+          itemIndex={-1}
+          selection={props.selection}
+          selectionMode={(props.selection && props.selection.mode) || SelectionMode.none}
+          viewport={props.viewport}
+        />
+      </div>
+    </Sticky>
+  );
+}
+
+storiesOf('Sticky breadcrumb and sticy details list header', module)
+  .addDecorator(FabricDecorator)
+  .addDecorator(story => (
+    <Screener
+      steps={new Screener.Steps()
+        .snapshot('default', { cropTo: '.testWrapper' })
+        .executeScript("document.getElementsByClassName('ms-ScrollablePane--contentContainer')[0].scrollTop = 2")
+        .snapshot('scroll down by a small amount so that the first row is still visible and the header becomes sticky', { cropTo: '.testWrapper' })
+        .executeScript("document.getElementsByClassName('ms-ScrollablePane--contentContainer')[0].scrollTop = 99999")
+        .snapshot('scroll down to the bottom', { cropTo: '.testWrapper' })
+        .executeScript("document.getElementsByClassName('ms-ScrollablePane--contentContainer')[0].scrollTop = 0")
+        .snapshot('scroll up to the top', { cropTo: '.testWrapper' })
+        .executeScript("document.getElementsByClassName('ms-ScrollablePane--contentContainer')[0].scrollLeft = 10")
+        .snapshot('scroll left', { cropTo: '.testWrapper' })
```

Comment:
Isn't this test scrolling to the "right" in LTR? Confused me when looking at the screener output.